example.config.yaml

```diff
@@ -0,0 +1,12 @@
+ledger: /home/jenr/.mina-indexer/mainnet.json
+non_genesis_ledger: false
+root_hash: 3NKeMoncuHab5ScarV5ViyF16cJPT4taWNSaTLS64Dp67wuXigPZ
+startup_dir: /home/jenr/.mina-indexer/startup-blocks
+watch_dir: /home/jenr/.mina-indexer/watch-blocks
+database_dir: /home/jenr/.mina-indexer/database
+log_dir: /home/jenr/.mina-indexer/logs
+keep_non_canonical_blocks: true
+log_level: info
+log_level_stdout: trace
+prune_interval: 10
+canonical_update_threshold: 2
```

Comment:
Can you use env variables? s/jenr/$HOME/g

Reply:
No, the paths are parsed directly into `PathBuf`s. We'd need to write functionality to iterate through a `PathBuf` and replace `$ENV` patterns with their values using `std::env::var`. Cool feature, but it shouldn't block the merge of this PR

https://app.asana.com/0/1203669416123378/1205272524183351/f

Reply:
also what in the world is s//g

Reply:
global search and replace

Reply:
@trevorbernard